lib/gui/app/components/source-selector/source-selector.tsx

```diff
@@ -363,15 +518,9 @@ export class ImageSelector extends React.Component<
 							</>
 						) : (
 							<StepSelection>
-								<StepButton onClick={this.openImageSelector}>
-									Select image
-								</StepButton>
-								<Footer>
-									{mainSupportedExtensions.join(', ')}, and{' '}
-									<Underline tooltip={extraSupportedExtensions.join(', ')}>
-										many more
-									</Underline>
-								</Footer>
+								{_.map(this.flows, flow => {
```

Comment:
No need to define `this.flows` and use `_.map`, just use `<FlowSelector />` two times.

Reply:
There's no need to but it will look better with all the other flows implemented which we'll have relatively soon, and it costs nothing while reducing the clutter

Reply:
It won't look better.
You're defining a public attribute that is only used in one place in this class.